src/types/Server.tsx

```diff
@@ -1,10 +1,9 @@
-
 export enum ServerAction {
   UNKNOWN_ACTION = 'Unknown',
 
-  LOGIN = 'Login',
-  LOGOUT = 'Logout',
-  PASSWORD_RESET = 'Reset',
+  LOGIN = 'RestLogin',
```

Comment:
I removed the Rest everywhere finally.
I was not provided on all the server's type.
Let the values without Rest for the time being.

Reply:
Ok, fixed